### PR TITLE
Mobile structure

### PR DESCRIPTION
**Pull request by deadlight**
## Done

 - Added new site navigation for /mobile
 - Created stub pages for all of the mobile sections
 - Updated redirects to point /phone and /tablet URLs to /mobile
 - Removed tablet section
 - Added /mobile into full-width css configuration sections

## QA

 - go to /mobile and see that it exists as a contentless stub
 - check the secondary nav and that all of the sections work
 - go to /phone or /tablet and see the redirect to /mobile
 - Try some deeper pages from live ubuntu.com/phone or /tablet and check for sensible redirects.

### CODE REVIEW

File: static/css/section/_local.scss [hunk 1]
```diff
@@ -249,7 +249,8 @@ body {
 .server,
 .support,
 .tablet,
```
Comment (anthonydillon) [inline static/css/section/_local.scss]: We wont need `tablet` and `phone` here will we?

File: static/css/section/_local.scss [hunk 2]
```diff
@@ -298,7 +299,8 @@ body {
 .core,
 .server,
 .support,
-.tablet {
+.tablet,
```
Comment (anthonydillon) [inline static/css/section/_local.scss]: Lets tidy up the scss when we can by removing `tablet` and `phone` here.

File: static/css/section/_navigation.scss [hunk 3]
```diff
@@ -294,7 +294,8 @@
 .server,
 .support,
 .tablet,
```
Comment (anthonydillon) [inline static/css/section/_navigation.scss]: Please remove old sections.

File: static/css/section/_navigation.scss [hunk 4]
```diff
@@ -310,7 +311,8 @@
 .server,
 .server,
 .support,
-.tablet {
+.tablet,
```
Comment (anthonydillon) [inline static/css/section/_navigation.scss]: Here too.

File: templates/mobile/base_mobile.html [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
...m:rotate(0deg);}to{transform:rotate(-180deg);}}@-webkit-keyframes "rotateRev"{from{-webkit-transform:rotate(-180deg);}to{-webkit-transform:rotate(0);}}@-moz-keyframes rotateRev {from{-moz-transform:rotate(-180deg);}to{-moz-transform:rotate(0);}}@keyframes "rotateRev"{from{transform:rotate(-180deg);}to{transform:rotate(0);}}@-webkit-keyframes "animatedBackground"{0%{-webkit-transform:scale(1, 1);-moz-transform:scale(1, 1);-ms-transform:scale(1, 1);-o-transform:scale(1, 1);transform:scale(1, 1);opacity:0;}100%{-webkit-transform:scale(1.1, 1.1);-moz-transform:scale(1.1, 1.1);-ms-transform:scale(1.1, 1.1);-o-transform:scale(1.1, 1.1);transform:scale(1.1, 1.1);opacity:1;}}@-moz-keyframes animatedBackground {0%{-moz-transform:scale(1, 1);transform:scale(1, 1);opacity:0;}100%{-moz-transform:scale(1.1, 1.1);transform:scale(1.1, 1.1);opacity:1;}}@keyframes "animatedBackground"{0%{-webkit-transform:scale(1, 1);-moz-transform:scale(1, 1);-ms-transform:scale(1, 1);-o-transform:scale(1, 1);transform:scale(1, 1);opacity:0;}100%{-webkit-transform:scale(1.1, 1.1);-moz-transform:scale(1.1, 1.1);-ms-transform:scale(1.1, 1.1);-o-transform:scale(1.1, 1.1);transform:scale(1.1, 1.1);opacity:1;}}</style>
```
Comment (anthonydillon) [inline templates/mobile/base_mobile.html]: Do we need these keyframes in the new section?

File: templates/mobile/developers.html [hunk 6]
```diff
@@ -0,0 +1,27 @@
+{% extends "mobile/base_mobile.html" %}
+
+{% block title %}Ubuntu for phones and tablets{% endblock %}
```
Comment (anthonydillon) [inline templates/mobile/developers.html]: I imagine you will update the titles and descriptions when the content is available. Might be an idea to remove the content for now so its not missed when going live.

File: templates/mobile/base_mobile.html (hunk 5 above)
Comment (deadlight) [inline templates/mobile/base_mobile.html]: Yeah, we're reusing some of the animations. I'll make sure to audit it when the content is complete.

**Pull request merged**